projector-launcher/build.gradle.kts

```
@@ -65,9 +67,32 @@ dependencies {
 
 version = "1.1.0"  // npm doesn't support versions like "1.0-SNAPSHOT"
 
-val npmCommand = when (DefaultNativePlatform.getCurrentOperatingSystem().isWindows) {
-  true -> listOf("cmd.exe", "/C", "npm.cmd")
-  false -> listOf("npm")
+val kotlinNodeJsSetupTask = rootProject.tasks.getByName<NodeJsSetupTask>("kotlinNodeJsSetup")
```

Why do you think this variant is better than the last one I've suggested? Fmpov, it's good to have the name of the task only a single time here.

```suggestion
val kotlinNodeJsSetup: NodeJsSetupTask by rootProject.tasks
```